src/App.tsx

```diff
@@ -1,4 +1,9 @@
 import { useState, useEffect } from "react";
+import  Navbar  from "./Navbar";
+import { BrowserRouter, Routes, Route, HashRouter } from 'react-router-dom';
+import Create from "./Create";
+import Home from "./Home"
+import Reports from "./Reports";
```

Comment:
this is more of my preference so not something that needs fixing. I have started to separate out what is imported as a module vs code that I write. so this would become:

```js
import { useState, useEffect } from "react";
import { Routes, Route, HashRouter } from "react-router-dom";

import Navbar from "./Navbar";
import Create from "./Create";
import Home from "./Home";
import Reports from "./Reports";
```